cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,14 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
```

Comment:
@gbraad 
> $ minishift hostfolder setup-users

With this the `Users` directory will be mounted to some location prefix with `/mnt/sda` right?